index/elasticsearch.py

```diff
@@ -100,7 +101,13 @@ def build_query(
         self.build_match_query(query, keywords)
         query = {"query": {"bool": query}}
         self.add_pagination_fields(query, offset, size)
-        self.build_sort_query(query)
+
+        if sort_by == "date_by_descending":
+            self.build_sort_query(query, "desc")
+        elif sort_by == "date_by_ascending":
+            self.build_sort_query(query, "asc")
```

Comment:
```suggestion
        if sort_by == "descending_date":
            self.build_sort_query(query, "desc")
        elif sort_by == "ascending_date":
            self.build_sort_query(query, "asc")
```

I think these names are little better.